packages/model-generator/src/s3_string_object_fetcher.ts

```diff
@@ -14,15 +15,28 @@ export class S3StringObjectFetcher {
    */
   fetch = async (uri: string) => {
     const { bucket, key } = this.parseS3Uri(uri);
-    const getSchemaCommandResult = await this.s3Client.send(
-      new GetObjectCommand({ Bucket: bucket, Key: key })
-    );
-    const schema = await getSchemaCommandResult.Body?.transformToString();
-    if (!schema) {
-      // eslint-disable-next-line amplify-backend-rules/prefer-amplify-errors
-      throw new Error('Error on parsing output schema');
+    try {
+      const getSchemaCommandResult = await this.s3Client.send(
+        new GetObjectCommand({ Bucket: bucket, Key: key })
+      );
+      const schema = await getSchemaCommandResult.Body?.transformToString();
+      if (!schema) {
+        // eslint-disable-next-line amplify-backend-rules/prefer-amplify-errors
```

Comment:
If the lint rule specifically asks for `AmplifyUserError` then it's an issue. Any `Amplify` error should be fine. Not for this PR, but if that's the case we should create a backlog tech debt item to fix it.

Reply:
Lint might be fine with `AmplfiyFault` (will check), this is a relic of making a version with `Error`